src/PKI.tsx

```diff
@@ -145,42 +285,189 @@ const PKI: React.FC<{ kmsToken: string }> = ({ kmsToken }) => {
           <HeadingWithCode heading="Data encrypted under Client 1’s key" />
           <Stack spacing={3}>
             <Text>
-              <Text as="b">Client 1</Text> and <Text as="b">Client 2</Text> has allready done theses actions:
+              <ClientBadge client={1}>Client 1</ClientBadge> and <ClientBadge client={2}>Client 2</ClientBadge> has allready done theses
+              actions:
             </Text>
             <UnorderedList>
               <ListItem>
-                <Text as="b">Client 1</Text> has generated Key Pair
+                <ClientBadge client={1}>Client 1</ClientBadge> has generated Key Pair <Code>sk_1/pk_1</Code>
```

Comment:
he does not really create a random key pair - but CoverCrypt Policy, masterKeyPairs and userDecryptionKey associated